migtests/tests/pg/views-and-rules/pg_views_and_rules_automation.sql

```diff
@@ -56,6 +56,7 @@ select * from test_views.v2;
 
 create or replace view test_views.v3 as select a.first_name,b.last_name from test_views.view_table1 a inner join test_views.view_table2 b using(id);
 
+create or replace view test_views.v4 as select (a.first_name || ' ' || a.last_name || ';'::text) as full_name from test_views.view_table1 a;
```

Comment:
@makalaaneesh this is a single line DDL which i think would have worked already.
Add a test cases like this where one of the lines has `;` in it.

```
CREATE OR REPLACE VIEW view test_views.v4 as 
SELECT 
 (a.first_name || ' ' || a.last_name || ';'::text) as full_name 
 from test_views.view_table1 a;
```

```
 SELECT l._assay_key,
    l.sequencenum,
    g._gellanestructure_key,
    g._gellane_key,
    g._emapa_term_key,
    g._stage_key,
    g.creation_date,
    g.modification_date,
    s.term,
    t.stage,
    ((('TS'::text || ((t.stage)::character varying(5))::text) || ';'::text) || s.term) AS displayit
```

Reply:
@sanyamsinghal when pg_dump exports schema, it comes as multi-line. So, for example,view.sql for this statement is: 
```
CREATE VIEW test_views.v4 AS
 SELECT ((((a.first_name)::text || ' '::text) || (a.last_name)::text) || ';'::text) AS full_name
   FROM test_views.view_table1 a;
   ```
   
  This covers the multi-line case

Reply:
`);` is the base case (every table is output like this in pg_dump), so it is already covered.